CodeEdit/NavigatorSidebar/SourceControlNavigator/Views/Commits/ProjectCommitHistoryView.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  ProjectCommitHistoryView.swift
+//  CodeEdit
+//
+//  Created by Nanashi Li on 2022/05/04.
+//
+
+import SwiftUI
+
+struct ProjectCommitHistoryView: View {
+    var body: some View {
+        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
+    }
+}
```

Comment:
Remove placeholder values

Reply:
It's a test editor view to see if the editor opens it, and it's not the final version.